Doesn't compile when used by Custom.h

I didn’t change anything in the custom.h file, I just renamed the custom-sample.h

src\src\Helpers_tmp\__tmpfile.cpp:29280:103: error: array subscript out of bound
 constexpr size_t Highest_CPlugin_id = ProtocolIndex_to_CPlugin_id[ProtocolIndex_to_CPlugin_id_size - 1];
                                                                                                       ^
src\src\Helpers_tmp\__tmpfile.cpp:29287:77: error: size of array 'CPlugin_id_to_ProtocolIndex' is not an integral constant-expression
 protocolIndex_t CPlugin_id_to_ProtocolIndex[CPlugin_id_to_ProtocolIndex_size]{};
                                                                             ^
Compiling .pio\build\custom_ESP8266_4M2M\FrameworkArduino\FS.cpp.o
Compiling .pio\build\custom_ESP8266_4M2M\FrameworkArduino\FSnoop.cpp.o
src\src\Helpers_tmp\__tmpfile.cpp: In function 'cpluginID_t getHighestIncludedCPluginID()':
src\src\Helpers_tmp\__tmpfile.cpp:29327:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
cc1plus.exe: some warnings being treated as errors
*** [.pio\build\custom_ESP8266_4M2M\src\src\Helpers_tmp\__tmpfile.cpp.o] Error 1
======================================================================================== [FAILED] Took 40.87 seconds ========================================================================================
Environment          Status    Duration
-------------------  --------  ------------
custom_ESP8266_4M2M  FAILED    00:00:40.873
=================================================================================== 1 failed, 0 succeeded in 00:00:40.873 ===================================================================================
 *  The terminal process "C:\Users\big_a\.platformio\penv\Scripts\platformio.exe 'run', '--environment', 'custom_ESP8266_4M2M'" terminated with exit code: 1.

--- a/src/src/Helpers/_CPlugin_init.cpp
+++ b/src/src/Helpers/_CPlugin_init.cpp
@@ -2070,7 +2070,7 @@ const CPlugin_ptr_t PROGMEM CPlugin_ptr[] =
 constexpr size_t ProtocolIndex_to_CPlugin_id_size = sizeof(ProtocolIndex_to_CPlugin_id);
 
 // Highest CPlugin ID included in the build
-constexpr size_t Highest_CPlugin_id = ProtocolIndex_to_CPlugin_id[ProtocolIndex_to_CPlugin_id_size - 1];
+constexpr size_t Highest_CPlugin_id = ProtocolIndex_to_CPlugin_id_size == 0 ? 0 : ProtocolIndex_to_CPlugin_id[ProtocolIndex_to_CPlugin_id_size - 1];
 
 constexpr size_t CPlugin_id_to_ProtocolIndex_size = Highest_CPlugin_id + 1;
 

